# ====== DO NOT MODIFY! AUTOGENERATED FROM AN ACTION DEFINITION ======
#result definition
uint8 extra_info
bool achieved_goal 		#if true, the goal has been achieved
